sdk/scheduler/src/test/java/com/mesosphere/sdk/state/StateStoreUtilsTest.java

```diff
@@ -32,6 +33,8 @@
  */
 public class StateStoreUtilsTest {
```

Comment:
Missing unit tests against the new StateResource calls?

Should also exercise the malicious large content cases described above:
- `Content-Length` too big: fail fast.
- `Content-Length` small but actual content too big: fail during streaming.

Reply:
(These cases could probably be done by mocking InputStream and/or FormDataContentDisposition)